viite-UI/src/model/RoadAddressProjectCollection.js

```diff
@@ -336,6 +361,11 @@
       return dirtyProjectLinks;
     };
 
+    this.setCurrentRoadPartList = function(parts){
+      currentRoadPartList = parts.slice(0);
+      dirtyRoadPartList = parts.slice(0);
```

Comment:
The dirty is saving all temporary changes in UI and is the list that the user can see. The currentRoadPartList is used when user presses Peruuta and wants to discard all changes in dirty list